habitat_baselines/rl/ddppo/policy/resnet_policy.py

```diff
@@ -263,12 +289,44 @@ def forward(self, observations, rnn_hidden_states, prev_actions, masks):
             visual_feats = self.visual_fc(visual_feats)
             x.append(visual_feats)
 
-        tgt_encoding = self.get_tgt_encoding(observations)
         prev_actions = self.prev_action_embedding(
             ((prev_actions.float() + 1) * masks).long().squeeze(-1)
         )
+        x.append(prev_actions)
```

Comment:
This needs to be done after the block for `IntegratedPointGoalGPSAndCompassSensor.cls_uuid`, otherwise the pre-trained pointnav models won't work.